pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -23,21 +23,30 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
-const parallelism = 16
+var (
+	parallelism = 16
+)
+
+// SetParallelism sets the parallelism for all scheduler algorithms.
```

Comment:
add a `TODO(name or issue)` here to move to framework and drop the package var, and indicate in a godoc comment this should only be called once per scheduler process